### PR TITLE
[release] News update for the next release (1.12)

### PR DESCRIPTION
**Pull request by valeriuo**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -9,6 +39,96 @@ Noteworthy changes in release a.b
 * The old htslib/knetfile.h interfaces have been marked as deprecated.  Any
   code still using them should be updated to use hFILE instead. (#1200)
 
+* Added an introspection API for checking some of the capabilities provided
+  by HTSlib. (#1170)
+    - `hfile_list_schemes`: returns the number of schemes found
+    - `hfile_list_plugins`: returns the number of plugins found
+    - `hts_has_plugin`: checks if a specific plugin is available
```
Comment (jmarshall) [inline NEWS:46]:
```suggestion
    - `hfile_has_plugin`: checks if a specific plugin is available
```

File: NEWS [hunk 2]
```diff
@@ -9,6 +39,104 @@ Noteworthy changes in release a.b
...
+  intentional fall through case statements, when building with 
+  `-Wextra flag`. Thanks to John Marshall. (#1163)
+
+
+Bug fixes
+---------
+
+* Fixed a bug in the VCF header parser, which prevent it from reading samples
+  names containing UTF-8 characters. Thanks to Sam Morris for reporting the
+  issue. (#1237; fixed samtools/bcftools#1408)
```
Comment (jmarshall) [inline NEWS:106]: I don't think _htslib/NEWS_ should state that UTF-8-encoded characters are valid in these sample names, as that is at best unclear in the VCF spec. Suggest describing this fix as something like “Fixed VCF `#CHROM` header parsing to only separate columns at tab characters” — not that that's completely true: actually this hacky code splits at any non-`\n` control character. 🤮  (Also “samples names” typo.)

Comment (valeriuo) [reply NEWS]: Done. Thanks!

**Comment by daviesrob** [comment]
Added a very small adjustment prior to squash + merge.

File: NEWS [hunk 3]
```diff
@@ -60,6 +60,11 @@ Features and Updates
 * Added two public methods, `sam_parse_cigar` and `bam_parse_cigar`, as part of
   a small CIGAR API (#1169, #1182). Thanks to Daniel Cameron for input. (#1147)
 
+* HTSlib, and the included htsfile program, will now recognise the old
+  RAZF compressed file format.  Note that while the format is detected,
+  HTSlib is unable to read it.  It is recommended that RAZF files are
+  uncompressed with `gunzip` before using them with HTSlib.  Thanks to
+  John Marshall. (#1244)
```
Comment (jmarshall) [inline NEWS:67]: Probably should also credit the reporter — Matthew J. Oldach and samtools/samtools#1387 — who actually managed to encounter a RAZF file.

**Pull request merged**